continuous_integration/gpuci/environment.yaml

```diff
@@ -0,0 +1,17 @@
+name: gpuci
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - nvidia
+dependencies:
+  - rust>=1.60.0
```

Comment:
Should we target these changes for the datafusion branch?

Reply:
That would make sense, though on gpuCI's end we would need to grab this file from one of the branches to pull in during image builds ([example of this being done with the environment file](https://github.com/rapidsai/dask-build-environment/blob/8da1d032d855d487f760749f8bffeb1e532caea9/dask_sql.Dockerfile#L14)), and it just seemed to make more sense to target the `main` for this.

If we want to have different versions of this file in `main` and the datafusion branch, that might motivate using separate images for gpuCI running on datafusion PRs vs. PRs into `main`, though not sure if that's worth the extra builds since the images only differ by a handful of added packages

Reply:
That makes sense w.r.t CI. It might also make sense to include this file (in a separate PR) in the datafusion branch even if we don't use it for gpuCI since one of the motivations for the file was for users working on that branch to be able to setup gpu environments to run tests locally